[INFO ] 2019-01-27T10:55:12,302Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105509.627/NotaryService {}
[INFO ] 2019-01-27T10:55:12,767Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:55:13,307Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T10:56:33,564Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105631.532/NotaryService {}
[INFO ] 2019-01-27T10:56:33,901Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:56:34,429Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T10:57:09,281Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105707.155/NotaryService {}
[INFO ] 2019-01-27T10:57:09,603Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:57:10,132Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T10:58:20,234Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105818.055/NotaryService {}
[INFO ] 2019-01-27T10:58:20,568Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:58:21,106Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T10:58:42,452Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105840.410/NotaryService {}
[INFO ] 2019-01-27T10:58:42,810Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:58:43,343Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T10:59:16,348Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-105914.338/NotaryService {}
[INFO ] 2019-01-27T10:59:16,700Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T10:59:17,237Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T11:01:09,912Z [driver-pool-thread-0] network.NodeInfoFilesCopier.addConfig - Now watching: /Previous Content/Users/nguyenthanhbinh/Blockchain/test/corda/tools/explorer/build/node-driver/20190127-110108.078/NotaryService {}
[INFO ] 2019-01-27T11:01:10,258Z [driver-pool-thread-0] internal.DriverDSLImpl.startOutOfProcessNode - Starting out-of-process Node Notary Service, debug port is not enabled {}
[ERROR] 2019-01-27T11:01:10,785Z [main] internal.DriverDSLImpl.genericDriver - Driver shutting down because of exception [errorCode=10lckx4, moreInformationAt=https://errors.corda.net/OS/5.0-SNAPSHOT/10lckx4] {}
java.lang.IllegalStateException: Unable to start notaries. A required port might be bound already.
	at net.corda.testing.node.internal.DriverDSLImpl.start(DriverDSLImpl.kt:372) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.DriverDSLImplKt.genericDriver(DriverDSLImpl.kt:964) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.driver.Driver.driver(Driver.kt:177) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.explorer.ExplorerSimulation.startDemoNodes(ExplorerSimulation.kt:73) ~[main/:?]
	at net.corda.explorer.ExplorerSimulation.<init>(ExplorerSimulation.kt:63) ~[main/:?]
	at net.corda.explorer.MainKt.main(Main.kt:136) ~[main/:?]
Caused by: net.corda.testing.node.internal.ListenProcessDeathException: The process that was expected to listen on localhost:20000 has died with status: 1
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt:155) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$addressMustBeBoundFuture$1.invoke(InternalTestUtils.kt) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at net.corda.testing.node.internal.InternalTestUtilsKt$poll$task$1.run(InternalTestUtils.kt:202) ~[corda-node-driver-5.0-SNAPSHOT.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_191]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
[INFO ] 2019-01-27T11:02:37,519Z [JavaFX Application Thread] model.NodeMonitorModel.establishConnectionWithRetry - Connecting to: :0 {}
[INFO ] 2019-01-27T11:02:38,228Z [JavaFX Application Thread] internal.RPCClient.logElapsedTime - Failed Startup took 672 msec {}
[INFO ] 2019-01-27T11:05:38,556Z [JavaFX Application Thread] model.NodeMonitorModel.establishConnectionWithRetry - Connecting to: localhost:10003 {}
[INFO ] 2019-01-27T11:05:39,089Z [JavaFX Application Thread] internal.RPCClient.logElapsedTime - Failed Startup took 532 msec {}
